labwork5/Regular expressions.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+# task 1.1
+text = 'gswdhc@epa.com djfbgh fdfivgidebvfu@mail.ru vbdsbvj sonskvn@gmail.com'
+print(re.findall(r'\w+@\w+\.\w+', text))
+
+
+# task 1.2
+text = 'wreyru erfh elfjrlfje ldfl nfn fn'
+all_words = re.findall(r'\w+', text)
+for word in all_words:
+    if len(word) in (3, 4, 5):
+        print(word)
+
+
+# task 1.3
+text = 'wreyru erfh elfjrlfje ldfl nfn fn6 7 86 340'
+print(re.findall(r'\d+', text))
```

Comment:
Just a tip. You can also add int(number) to delete leading zero: i.e. if it would be 's086sss' your script would print 086 but we print numbers according to task conditions, not digits and it would be more beautiful to print 86.